C/Find_Max_Min/Max_Min.c

```diff
@@ -0,0 +1,18 @@
+// C program to find maximum and minimum of
+// two numbers without using loop and any
+// condition.
+#include<bits/stdc++.h>
```

Comment:
Please replace` #include<bits/stdc++.h>` with `#include<stdlib.h>`

Reply:
Ok, I did it, thanks for helping! :)